src/components/blocks/CampusWeatherDetail/CampusWeatherCurrentConditions/CampusWeatherCurrentConditions.tsx

```diff
@@ -0,0 +1,21 @@
+import { celsiusToFahrenheit, getCompassDirection, kphToMph } from '@/util/campusUnitConversion'
+import styles from './CampusWeatherCurrentConditions.module.scss'
+import { CampusWeatherCurrentValue } from './CampusWeatherCurrentValue/CampusWeatherCurrentValue'
+
+export const CampusWeatherCurrentConditions = ({ currentWeatherData }) => {
+	const { temperature, dewpoint, windDirection, windSpeed, icon, relativeHumidity, windChill, heatIndex, textDescription } = currentWeatherData
+	const apparentTemperature = windChill.value !== null ? windChill.value : heatIndex.value !== null ? heatIndex.value : temperature.value
```

Comment:
Is this something we could make more intuitive to read + make into a utility?

Reply:
commit ID: 3f38a93cbda351554f21e83c2e7ca2a55d191b4c

Addressed intuitiveness by replacing nested ternary conditions with a switch statement. Unsure of utility potential; wind chill and heat index arrive either null or pre-calculated from the api response. They will never both be set, and if both are null then apparent temp matches air temperature. A utility to calculate heat index or wind chill may be useful, for sure, but we don't have to do that here.

Reply:
commit ID: b8db955c79732b977a76a6368ed3508ccc95dd09

Fixed dumdum mistake.